app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -13,6 +13,10 @@
 
         h2.govuk-heading-l = t("jobs.applying_for_the_job")
 
+        = govuk_notification_banner title: t("banners.important") do |banner|
+          - banner.add_heading text: t(".notification.heading")
+          .govuk-body = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
```

Comment:
You don't need any element at all here, if you do then the banner will have the extra margin